samples/sql_allowed_authorized_networks.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+apiVersion: constraints.gatekeeper.sh/v1alpha1
+kind: GCPSQLAllowedAuthorizedNetworksConstraintV1
+metadata:
+  name: sql_allowed_authorized_networks_whitelist
+spec:
+  severity: high
+    
```

Comment:
nit. Let's drop this empty line.

Reply:
fixed here and in the original test constraint